### PR TITLE
no-issue: PR template for releases

### PR DESCRIPTION
**Pull request by jaskfla**
### Changes

Created a PR template for releases. After merging, this can (in theory) be invoked using https://github.com/beyondessential/tupaia/compare/master...dev?quick_pull=1&title=Release+20yy-w&labels=release&template=release_template.md, which will also automatically add the **release** tag.

If an open `master` ← `dev` PR already exists, GitHub will prompt you to view that one instead of creating a new one.

The [existing pull request template](https://github.com/beyondessential/tupaia/blob/dev/.github/pull_request_template.md) remains the default. Unfortunately, this needs to stay at the top level of the `./github` folder; moving it into the new `./github/PULL_REQUEST_TEMPLATE` folder makes it accessible only via a link with a `template` [query parameter](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/using-query-parameters-to-create-a-pull-request) (like the one above).

### Testing

This can’t be comprehensively tested until it’s merged into the default branch. Instead, I replicated this template setup in a private repo on my own account, and it seems to work there

---

### Screenshots

None.

### CODE REVIEW

**Comment by jaskfla** [comment]
After merging, I’ll add a bookmark to the big ugly link to the releases Slack channel

File: .github/PULL_REQUEST_TEMPLATE/release_template.md
Comment (edmofro) [inline .github/PULL_REQUEST_TEMPLATE/release_template.md:7]: Have we used this section in recent history? Can probably drop it, as visualisations have become Maui responsibility and configured via the front end, rather than dev work.

File: .github/PULL_REQUEST_TEMPLATE/release_template.md
Comment (edmofro) [inline .github/PULL_REQUEST_TEMPLATE/release_template.md:3]: I recall a section "Tweaks" that is missing from this list. Perhaps there was a decision to consolidate tweaks into the features section, but it could also be that we just didn't use it in the PR you've taken this from

**Review by edmofro — CHANGES_REQUESTED** [review]
Thanks for taking initiative here Jasper, awesome

File: .github/PULL_REQUEST_TEMPLATE/release_template.md
Comment (jaskfla) [reply .github/PULL_REQUEST_TEMPLATE/release_template.md:7]: I only had a look through the most recent six releases (ignoring ones that were closed without merging), and it was used in 2 of those 6. Do you reckon remove it? I’ll defer to your call here

Comment (edmofro) [reply .github/PULL_REQUEST_TEMPLATE/release_template.md:7]: Can you link me to those where it was used?

File: .github/PULL_REQUEST_TEMPLATE/release_template.md
Comment (jaskfla) [reply .github/PULL_REQUEST_TEMPLATE/release_template.md:3]: You recall correctly! Though it looks like the last time we had a **Tweaks** section was around this time last year: #4267

Comment (edmofro) [reply .github/PULL_REQUEST_TEMPLATE/release_template.md:3]: I like it, personally. Separates the wheat from the chaff.

File: .github/PULL_REQUEST_TEMPLATE/release_template.md
Comment (jaskfla) [reply .github/PULL_REQUEST_TEMPLATE/release_template.md:7]: These are from the last 10 releases:

- #4921
- #4697
- #4360

Comment (edmofro) [reply .github/PULL_REQUEST_TEMPLATE/release_template.md:11]: Yeah I suspected it'd be like that - those aren't visualisations, they're "features" and "tweaks" to the types of visualisations we can display. Historically, an entry would look like "Malaria cases per month", etc. The section can definitely go.

**Pull request merged**
